pallets/parachain-staking/src/set.rs

```diff
@@ -38,18 +38,20 @@ impl<T: Ord> OrderedSet<T> {
 		Self(Vec::new())
 	}
 
-	/// Create a set from a `Vec`.
+	/// Create an ordered set from a `Vec`.
 	///
-	/// `v` will be sorted and dedup first.
+	/// The vector will be sorted reversily (from greatest to lowest) and
+	/// deduped first.
 	pub fn from(mut v: Vec<T>) -> Self {
-		v.sort();
```

Comment:
I would perhaps rename this to `from_sorted_vec`, as it might not be clear whether the vec is assumed to be sorted or if it is sorted internally, even though there are comments. But I don't have a strong argument against keeping it like this.

Reply:
The input vector does not need to be sorted but will be sorted when creating the set?!

Reply:
Right... 😅